_layouts/post.html

```diff
@@ -8,7 +8,7 @@
   <div class="container">
     <div class="newsletter col-wide">
 
-      <a href="{{ site.baseurl }}/news" > < Back to news</a>
```

Comment:
This looks like an erroneous opening angle bracket but it is actually an intended literal angle bracket.
It would probably work better as a back arrow from one of the [icon packs](https://github.com/fortran-lang/fortran-lang.org/blob/master/CONTRIBUTING.md#icon-packs) but it shouldn't be removed here.